department-of-veterans-affairs/caseflow
# adding an additional check to see if sp can be finalized

Resolves #6522 

### Description
Adds another check for schedule period to see if it can be processed to submit to VACOLS

### Acceptance Criteria
- [ ] Test plan executed

### Testing Plan
1. Upload the RO spreadsheet for periods 10-01-2018 to 04-30-2019
2. Upload the same spreadsheet again for the same periods
3. Submit the 1st set of assignments to VACOLS
4. Try to submit the 2nd set of assignments to VACOLS
5. Notice it fail with an error
5. Try entering the URLs of the uploads i.e (http://localhost:3000/hearings/schedule/build/upload/1 and 2)
6. Page expiration needs to be shown to users

[ROAssignmentTemplate-final.xlsx](https://github.com/department-of-veterans-affairs/caseflow/files/2329798/ROAssignmentTemplate-final.xlsx)
